tools/js-optimizer.js

```diff
@@ -5887,6 +5887,73 @@ function safeHeap(ast) {
   });
 }
 
+function growableHeap(ast) {
```

Comment:
should this have some comment like "transform HEAP8 etc heap accesses to calls to GROWABLE_HEAP_LOAD_I8" or something?

Reply:
Adding a comment.